samples/AspNetWebApp/Utils.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace AspNetWebApp;
+
+static class Utils
+{
+    public static async Task ParallelForEachAsync<T>(this IEnumerable<T> items, int maxConcurrency, Func<T, Task> action)
```

Comment:
C# has a `Parallel` helper already. I recommend using that:

https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.parallel.foreach?view=net-9.0

Reply:
deleted